imio/dms/mail/migrations/migrate_to_3_0.py

```diff
@@ -1426,6 +1426,8 @@ def update_config(self):
         # update actionspanel transitions config
         key = "imio.actionspanel.browser.registry.IImioActionsPanelConfig.transitions"
         values = api.portal.get_registry_record(key)
+        if "task.back_in_created2|" not in values:
```

Comment:
je n'ai en effet pas assez précisé. Tu peux remettre du code sous la partie "TEMPORARY TO 3.0.61".